### PR TITLE
feat: change footer background

### PR DESCRIPTION
**Pull request by realsh9dy**
the footer was modified

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -86,7 +86,7 @@ <h1 class="title">
   </a>
   <footer>
     <div class="foot">
-      Made with ❤️ by <a href="https://github.com/wasimreja">Wasim Reja</a>
+      Made with ❤️ by <a href="https://github.com/wasimreja"><b>Wasim Reja</b></a>
```
Comment (wasimreja) [inline index.html:89]: keep it as is

```suggestion
      Made with ❤️ by <a href="https://github.com/wasimreja">Wasim Reja</a>

```

File: style.css [hunk 2]
```diff
@@ -506,7 +506,7 @@ h1 {
 }
 
 .foot {
-	color: #737373;
+	color: #27231F;
```
Comment (wasimreja) [inline style.css:509]:
```suggestion
	color: #737373;
```

**Review by wasimreja — CHANGES_REQUESTED** [review]
set a variable `--footer-color` in the root and dark theme. 
in root set the footer color to `#f4f1f1` and in dark theme `#26272b`

File: style.css (hunk 2 above)
Comment (realsh9dy) [reply style.css:511]: the requested changes are made

**Comment by realsh9dy** [comment]
Done.

File: style.css [hunk 3]
```diff
@@ -84,7 +86,7 @@ footer {
   bottom: 0;
   margin-top: 1.5rem;
   width: 100%;
-  background-color: #26272b;
+  background-color: #fca567;
```
Comment (wasimreja) [inline style.css:89]:
```suggestion
  background-color: var(--footer-color);
```

**Review by wasimreja — COMMENTED** [review]
make all the necessary changes

File: index.html (hunk 1 above)
Comment (wasimreja) [reply index.html:89]: @realsh9dy

File: style.css (hunk 2 above)
Comment (wasimreja) [reply style.css:511]: @realsh9dy

**Comment by realsh9dy** [comment]
please check

**Pull request closed without merging**